debian/autobake-deb.sh

```diff
@@ -94,6 +101,7 @@ case "${LSBNAME}"
 in
   # Debian
   "buster")
+    switch_back_to_ncurses5
```

Comment:
these branches are fall thorough, so only a `bookworm` entry is required.

Reply:
Good point I'll sorted that out in next version

Reply:
and buster has libbcurses-dev as 6 - https://packages.debian.org/buster/libncurses-dev.

With bionic at EOL the switchback function can be removed along with the bionic case statement.

Lets rebase to 11.4 too.

Reply:
I've rebased and I'll remove it. Should Bionic be removed all along from from 10.4 or is there another build coming up?

Reply:
Sorry I did typo 11.4 meaning 10.4. Release is due out any day now, so a rebase and we'll just merge it after the release is out.

Reply:
Ok this should be rebased to 10.4 and remove from there. Ok I'll do it.